products/opensuse/main.pm

```diff
@@ -84,7 +84,11 @@ unless (get_var("DESKTOP")) {
 
 if (check_var('DESKTOP', 'minimalx')) {
     set_var("NOAUTOLOGIN", 1);
-    set_var("XDMUSED",     1);
+    # light is the default DM since Leap 15.0 per boo#1081760
+    if (!is_leap('15.0+')) {
```

Comment:
when it possible always better to use positive conditions :

`if(is_leap('<15.0'))`

Reply:
technically `(is_leap('<15.0'))` is not equals to `(!is_leap('15.0+'))`, if use `if(is_leap('<15.0'))` then breaks Tumbleweed